tfx_addons/huggingface_pusher/runner.py

```diff
@@ -34,6 +35,15 @@
 _DEFAULT_MODEL_VERSION_PLACEHOLDER_KEY = "$MODEL_VERSION"
 
 
+def _is_text_file(path):
```

Comment:
Let's move "Only the contents inside text based files should be replaced" to `_replace_placeholders_in_files` since that's where the replacing happens.

Reply:
good point! thank you!

Reply:
also nit: Let's add type annotations